Esri/arcgis-maps-sdk-swift-samples
# [Update] Address UI feedback from Nick

## Description

This PR updates various feedback from Nick on the initial testing for 200.1.0 sample viewer TestFlight app.

## Linked Issue(s)

- `swift/issues/3931`

## How To Test

- Verify all changed samples are still functionally same as before
- Double check if other screenshots need to be updated
